docs/core/common/assets/js/api/core/store.md

```diff
@@ -0,0 +1,128 @@
+## API - `$e.store`
+*  **Description**: `$e.store` API is a global state manager that allows to create global states for components.
+   It's actually a wrapper around [Redux Toolkit](https://redux-toolkit.js.org/introduction/getting-started), and it uses [Slices](https://redux-toolkit.js.org/api/createslice) under the hood.
+*  **Location**: *core/common/assets/js/api/core/store.js*
+*  **Methods**:
+
+| Method                        | Params                                                    | Returns                 | Description
+|-------------------------------|-----------------------------------------------------------|-------------------------|---------------------------------------------------------------------
+| `$e.store.register()`         | `{String}` *sliceId*, `{Slice}` *instance*   			    | `{void}`                | Register a new [Redux Toolkit Slice](https://redux-toolkit.js.org/api/createslice).
+| `$e.store.getAll()`           |                                                           | `{Object}`              | Get all existing Slices.
+| `$e.store.get()`              | `{String}` *sliceId* (optional)						    | `{Slice}` \| `{Object}` | Get a specific Slice instance by its ID, or get all of them if ID is empty.
```

Comment:
Why do we need `get all of them if ID is empty`?

Reply:
We need a way to get all slices. `getAll()` returns an array of strings (like the other JS APIs).
Do you want to add a `getAllSlices()`?

Reply:
Yes, please add a `getAllSlices` method.

Reply:
See #17787